ios/FluentUI/Popup Menu/PopupMenuController.swift

```diff
@@ -288,17 +294,21 @@ extension PopupMenuController: UITableViewDataSource {
         let section = indexPath.section
         let row = indexPath.row
         let item = sections[section].items[row]
-
-        let cell = tableView.dequeueReusableCell(withIdentifier: PopupMenuItemCell.identifier) as! PopupMenuItemCell
+        
```

Comment:
I think there are bunch of trailing whitespace violation in your changes(popupmenucontroller.swift, popupmenuitemcell, popupmenuprotocols.swift). Please make sure SwiftLint is installed and address all the format inconsistency.

Reply:
Done. Removed all the trailing spaces.